src/Behat/Context/FileContext.php

```diff
@@ -46,4 +46,40 @@ public function createFileWithName($filename, $directory = NULL) {
     $this->files[] = $this->getCore()->createFileWithName($path, $directory);
   }
 
-}
\ No newline at end of file
+  /**
+   * Get path for file in drupal 8.
+   *
+   * @param string $filename
+   *   The name of the file to get.
+   * @param string directory
+   *   A string containing the files scheme, usually "public://".
+   *
+   * @throws Exception
+   *   Exception file not found.
+   *
+   * @throws Exception
+   *   Exception file could not be copied.
+   *
+   * @Given file get name :filename
```

Comment:
Can we change the name of the step?
I propose:
```
Given I visit file with name :filename
Given I visit file with name :filename in directory :directory
```

I want know your opinion @rsanzante  about this.

Reply:
My opinion:

Given I visit file with name :filename
Given I visit file with name :filename in the :directory directory

Reply:
Better.